app/models/manageiq/providers/google/regions.rb

```diff
@@ -0,0 +1,50 @@
+module ManageIQ
+  module Providers::Google
+    module Regions
+      # From https://cloud.google.com/compute/docs/zones
+      REGIONS = {
```

Comment:
@lwander, any chance this data will eventually be available via the GCE API?  Not necessarily to consider for this PR, but the reason we have this data hard coded for AWS is that we don't currently (or, didn't when we wrote it) have a way to get the list of regions from AWS API.